packages/swapper/src/swappers/zrx/getZrxTradeQuote/getZrxTradeQuote.ts

```diff
@@ -23,32 +23,18 @@ export async function getZrxTradeQuote(
   if (buyAsset.chain !== ChainTypes.Ethereum || sellAsset.chain !== ChainTypes.Ethereum) {
     throw new ZrxError('getQuote - Both assets need to be on the Ethereum chain to use Zrx')
   }
-  if (!sellAmount && !buyAmount) {
-    throw new ZrxError('getQuote - sellAmount or buyAmount amount is required')
-  }
 
   const useSellAmount = !!sellAmount
   const buyToken = buyAsset.tokenId || buyAsset.symbol
   const sellToken = sellAsset.tokenId || sellAsset.symbol
 
-  const { minimum: minQuoteSellAmount, maximum: maxSellAmount } = await getZrxMinMax(input)
-
-  const minQuoteSellAmountWei = bnOrZero(minQuoteSellAmount).times(
-    bnOrZero(10).exponentiatedBy(sellAsset.precision)
-  )
+  const { minimum, maximum } = await getZrxMinMax(buyAsset, sellAsset)
 
-  const amount = useSellAmount ? { sellAmount } : { buyAmount }
-  const amountKey = Object.keys(amount)[0]
-  const amountValue = Object.values(amount)[0]
+  const minQuoteSellAmount = bnOrZero(minimum).times(bn(10).exponentiatedBy(sellAsset.precision))
 
-  let normalizedAmount = normalizeAmount(amountValue)
-  if (!normalizedAmount || normalizedAmount === '0') {
-    normalizedAmount = normalizeAmount(minQuoteSellAmountWei?.toString())
-  }
-
-  if (!normalizedAmount || normalizedAmount === '0') {
-    throw new ZrxError('getQuote - Must have valid sellAmount, buyAmount or minimum amount')
-  }
+  const normalizedSellAmount = normalizeAmount(
```

Comment:
kinda outside the scope of this PR, but we should update this utility function, the implementation is jank

https://github.com/shapeshift/lib/blob/a238e9656fe35017c13c4e6cb058a3240c447a79/packages/swapper/src/swappers/zrx/utils/helpers/helpers.ts#L48

if we updated the type signature to this, we could avoid this ternary style logic in consumption, and guarantee it returns something. a normalize function shouldn't return undefined 😕

`type normalizeAmount = (amount: string | number | BigNumber): string`